app/jobs/reports/base_report.rb

```diff
@@ -27,7 +27,7 @@ def public_bucket_name
     end
 
     def fiscal_start_date(time = Time.zone.now.beginning_of_day)
-      time.change(year: time.month >= 10 ? time.year : time.year - 1, month: 10, day: 1)
+      CalendarService.fiscal_start_date(report_date)
     end
```

Comment:
what if we removed this method from `BaseReport` here and inlined it at the callsite? `sp_active_users_report.rb`

Reply:
rubocop is also yelling `time` param is unused